src/main/kotlin/uk/gov/justice/digital/hmpps/prisontonhs/config/SqsConfig.kt

```diff
@@ -32,7 +31,6 @@ data class SqsConfigProperties(
 class SqsConfig {
 
   @Bean
-  @Primary
```

Comment:
this was the line causing the issue - meaning that both the normal queue and the dlq were using the same bean

Reply:
That suprises me you had to remove that since the beans have different names, unless a `@Qualifier` was missing somewhere it was rejected.

I thought we need `@primary` when spring boot cloud (or whatever it was called) auto-configed it's own AmazonSQS bean.

Oh well, if it works, it works

Reply:
When using localstack sharing a bean doesn't matter - it's only a problem when using AWS due to the access key and secret.  Probably need to revisit the audit api - it hasn't been tested properly against AWS yet.

Reply:
Yeah it does work for audit API on AWS, just tried the retry-dlq endpoint and it worked.  I think what Andy says about @Qualifier is probably the trick.